aWhereAPIs.json.postman_collection

```diff
@@ -1,442 +1,662 @@
-			"collectionId": "ebaad90a-3f21-61e4-7956-c1e23c7d610d",
-			"responses": []
-		},
-		{
-			"id": "dfff29c2-3afc-4bc5-7ce0-fd1872aa06ad",
-			"headers": "Authorization: Bearer {{aWhereAccessToken}}\n",
-			"url": "https://api.awhere.com/v2/fields/field1",
-			"pathVariables": {},
-			"preRequestScript": "",
-			"method": "GET",
-			"collectionId": "ebaad90a-3f21-61e4-7956-c1e23c7d610d",
-			"data": [],
-			"dataMode": "params",
-			"name": "Get a Single Field by ID",
-			"description": "####Request\nThis API call retrieves a single field by ID.\n\n[Get Fields Documentation](http://developer.awhere.com/api/reference/fields/get-fields)\n\n\n####Security\nThis API call uses the security Access Token that is retrieved with the \"Get a Token\" request. If you run that request first, it will save a token to Postman and this API will use it automatically. You can also see where the token should normally go by clicking the \"Headers\" tab below. The Authorization header holds the token, replacing the \"{{aWhereAccessToken}}\" part. \n",
-			"descriptionFormat": "html",
-			"time": 1445021824666,
-			"version": 2,
-			"responses": [],
-			"tests": "",
-			"currentHelper": "normal",
-			"helperAttributes": {},
-			"folder": "7fda93d1-ccfa-9816-95d8-9ca722fbd4f3"
-		},
-		{
-			"id": "e274458d-aee3-4aac-4e05-f8b15784f723",
-			"headers": "Authorization: Bearer {{aWhereAccessToken}}\n",
-			"url": "https://api.awhere.com/v2/weather/fields/field1/currentconditions",
-			"pathVariables": {},
-			"preRequestScript": "",
-			"method": "GET",
-			"collectionId": "ebaad90a-3f21-61e4-7956-c1e23c7d610d",
-			"data": [],
-			"dataMode": "params",
-			"name": "Current Conditions",
-			"description": "####Request\nThis API call provides a snapshot of recent weather using the weather station nearest to the field location.\n\n[Current Conditions Documentation](http://developer.awhere.com/api/reference/weather/current)\n\n_Tip: Remember to use a field ID that exists in your account. By default, this collection uses a default field ID of 'field1'_\n\n\n####Security\nThis API call uses the security Access Token that is retrieved with the \"Get a Token\" request. If you run that request first, it will save a token to Postman and this API will use it automatically. You can also see where the token should normally go by clicking the \"Headers\" tab below. The Authorization header holds the token, replacing the \"{{aWhereAccessToken}}\" part. \n",
-			"descriptionFormat": "html",
-			"time": 1445042373320,
-			"version": 2,
-			"responses": [],
-			"tests": "",
-			"currentHelper": "normal",
-			"helperAttributes": {},
-			"folder": "1f5f8325-8087-4816-20d0-0d3533cd53b8"
+			"request": {
```

Comment:
Besides the newer format, this is the main change.  The auth POST will now correctly use the environment variables.